src_new/constants.js

```diff
@@ -243,3 +243,12 @@ exports.SPECIAL_CASE_ID_PARTNERS = {
 		"params.syncTime": "number"
 	}
 }; //list of ID partners for whom special handling of datatype is required
+
+exports.ID_PARTNERS_CUSTOM_VALUES = {
+	"parrableId": [
+		{
+			"key": "params.timezoneFilter",
+			"value": {"allowedZones": [ "Pacific/Honolulu"]}
+		}
+	]
+}
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Missing semicolon.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=14890)

Reply:
Its the last object and on the last line in the file.